### PR TITLE
[DEV] Added project structure with tests and stubs

### PR DESCRIPTION
**Pull request by dmitry-uraev**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+beautifulsoup4==4.9.3
```
Comment (demid5111) [inline requirements.txt]: empty

File: requirements_qa.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+flake8==3.8.4
```
Comment (demid5111) [inline requirements_qa.txt]: keep pylint

File: scrapper.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+class Crawler:
+    """
+    Crawler implementation
+    """
+    def __init__(self, seed_urls: list, max_articles: int):
+        pass
+
+    @staticmethod
```
Comment (demid5111) [inline scrapper.py:31]: not sure we should leave it as static - let's discuss separately

File: pipeline.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    def __init__(self, corpus_manager: CorpusManager):
+        pass
+
+    def run(self):
+        """
+        Runs pipeline process scenario
+        """
+        pass
+
+    @staticmethod
```
Comment (demid5111) [inline pipeline.py:74]: the same - static? not sure

File: target_score.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Target score for scrapper.py:
+6
+
+# Target score for pipeline.py:
+6
+
+# Skip pipeline checks:
+1
```
Comment (demid5111) [inline target_score.txt:8]: ???

File: target_score.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Target score for scrapper.py:
```
Comment (demid5111) [inline target_score.txt:1]: why is the format so strange..

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (dmitry-uraev) [reply requirements.txt]: fixed

File: requirements_qa.txt (hunk 2 above)
Comment (dmitry-uraev) [reply requirements_qa.txt]: fixed

File: target_score.txt (hunk 5 above)
Comment (dmitry-uraev) [reply target_score.txt:8]: for future functionality only

File: target_score.txt (hunk 6 above)
Comment (dmitry-uraev) [reply target_score.txt:1]: it is simple for terminal commands

**Pull request merged**